include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -85,14 +86,12 @@ enum Flow { Source, Sink, Duplex };
 /// source is sink.  The reverse of sink is source.  The reverse of duplex is
 /// duplex.  The \p accumulatedFlow parameter sets the initial flow.  A user
 /// should normally \a not have to change this from its default of \p
-/// flow::Source.
-flow::Flow foldFlow(Value val, flow::Flow accumulatedFlow = flow::Source);
+/// Flow::Source.
+Flow foldFlow(Value val, Flow accumulatedFlow = Flow::Source);
```

Comment:
Is there any namespace guarding that we should add for `foldFlow`? E.g., put this in a `flow` namespace? I don't think you can add this to the `Flow` namespace (is that a namespace?)?

Reply:
it's in firrtl, is there a reason to restrict it more?

Reply:
The number of methods is pretty small here, but for direction, there's a bunch. See: https://github.com/llvm/circt/pull/992/commits/168e4b462f44f636dde85ade01fdce9df1505449#diff-e57b2a2bfee3df3e6d3a434153287680917019bb2f65a2d65ce44887e93b3153R27.

In Scala, I'd normally package all these utilities up in a common namespace to avoid prefixing/suffixing all the methods. More of a style nit in my own code.

I'd prefer if there was some way to do extension methods defined on `Fold`, but I don't think that's possible.